source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,57 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+Open a Change Stream
+--------------------
+
+You can keep track of changes of data in MongoDB, such as changes to a
+collection, database, or deployment by opening a :manual:`Change
+Stream</changeStreams/>`. A ``Change Stream`` allows applications to
+watch for changes to data and react to those changes. You can open a
+``Change Stream`` by calling the :node-api:`watch()
+</Collection.html#watch>` method on a collection, database, or
+deployment. The ``watch()`` method takes an optional pipeline as the
+first parameter that is an array of stages to filter the :manual:`change
+events </reference/change-events/>` output. 
+
+The ``watch()`` method can take in additional options as the second
+parameter. Create an `Object
+<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object>`_
+to specify additional options. Set ``fullDocument`` to
+``'updateLookup'`` to receive an event that contains the changes to the
+document as well as the entire altered document.
```

Comment:
Since there are only two accepted values for `fullDocument`, I wonder if it would be useful to contrast them here. Alternatively, you could reference the API docs with a link and skip the description of the `updateLookup` setting.

Reply:
How would you feel about the suggested additional sentence after this one: 

`
Alternatively, you can leave  ``fullDocument`` or set it to the string ``'default'`` to only receive the altered document, but not the changes to the document. 
`